pages/ui-components/packages.md

```diff
@@ -157,10 +156,9 @@ In webpack, include `includePaths` within the sassOptions of your Sass loader:
 ```js
 loader: "sass-loader",
 options: {
-  sourceMap: true,
   sassOptions: {
     includePaths: [
-      "./node_modules/@uswds",
+      "./node_modules/@uswds/uswds",
```

Comment:
```suggestion
```

Reply:
Removed in b05e3d4baddb9d5940f7ee911009897b7410b6b0